mkt/lookup/urls.py

```diff
@@ -47,7 +47,7 @@
         name='lookup.user_search'),
     url(r'^transaction_search$', views.transaction_search,
         name='lookup.transaction_search'),
-    url(r'^app_search\.json$', views.app_search,
+    url(r'^app_search\.json$', views.AppLookupSearchView.as_view(),
```

Comment:
nit: we could use the occasion to rename this? the `.json` part is a bit useless/misleading, the first time I had to do something in the lookup tool I thought it was a static file and was very confused for a couple minutes :)